<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-T8FKKRQ2" failures="2" tests="8" name="Test" time="78.455" errors="0" timestamp="2024-08-15T20:36:16 IST">
  <testcase classname="testCases.TC001_AccountRegistration" name="verify_Account_Registration" time="3.14"/>
  <testcase classname="testCases.TC002_LoginTest" name="loginTest" time="3.105"/>
  <testcase classname="testCases.TC003_LoginDataDrivenTest" name="LoginDataDrivenTest" time="2.208"/>
  <testcase classname="testCases.TC003_LoginDataDrivenTest" name="LoginDataDrivenTest" time="11.164"/>
  <testcase classname="testCases.TC003_LoginDataDrivenTest" name="LoginDataDrivenTest" time="11.178">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC003_LoginDataDrivenTest.LoginDataDrivenTest(TC003_LoginDataDrivenTest.java:50)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- LoginDataDrivenTest -->
  <testcase classname="testCases.TC003_LoginDataDrivenTest" name="LoginDataDrivenTest" time="14.487"/>
  <testcase classname="testCases.TC003_LoginDataDrivenTest" name="LoginDataDrivenTest" time="7.264"/>
  <testcase classname="testCases.TC003_LoginDataDrivenTest" name="LoginDataDrivenTest" time="2.318">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC003_LoginDataDrivenTest.LoginDataDrivenTest(TC003_LoginDataDrivenTest.java:62)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- LoginDataDrivenTest -->
</testsuite> <!-- Test -->
